packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -42,6 +42,12 @@
 
 .block-editor-panel-color-gradient-settings__dropdown-content .components-popover__content {
 	width: $sidebar-width;
+
+	.block-editor-color-gradient-control__tabs {
+		margin-top: - $grid-unit-10;
+		margin-left: - $grid-unit-10;
+		margin-right: - $grid-unit-10;
+	}
```

Comment:
I wonder if there's a better way to make sure that the Tabs fill the whole width of the sidebar — what about removing the padding from `.edit-site-screen-background-color__control` and adding it back only to the tab panel's contents?

Reply:
I suppose avoiding negative values has some merit, but both implementations would still be exposed to the same risk, no? Just trying to fully understand the implications :)

Reply:
It's not really about the negative margins, it's more about introducing custom style overrides of the `TabPanel` component.

Specifically to this case, please ignore my initial suggestion re. removing the padding from  — I mixed up this instance (which is rendered in a popover) with the one below (rendered in the sidebar).

It looks like the popover's contents have an `8px` margin added directly by the `Dropdown` component ([see the code here](https://github.com/WordPress/gutenberg/blob/8b1492131355aa6d42272faba25b841f0cd4509d/packages/components/src/dropdown/style.scss#L7)) (which I'm definitely not a fan of!)

![Screenshot 2022-07-05 at 21 13 55](https://user-images.githubusercontent.com/1083581/177399064-e52adfc6-7425-4a50-a286-8baf5c7be629.png)

In this specific situation, I'm not sure if there's a clean way to avoid the style override 🤔 

@mirka , any ideas?

Reply:
One solution that comes to mind is how the [`Card` component](https://wordpress.github.io/gutenberg/?path=/story/components-card--default) does it. It has a designated `<CardMedia>` component that you put inside `<Card>` when you want to break out of the padding. We could provide a similar subcomponent for `Dropdown`. ("Media" is not a very discoverable name for this, so ideally we'll give it a more literal name like `<DropdownContentFullWidth>` or something.)

I'm fine with the hacky style override as a temporary thing for this PR to land, and we can make a `DropdownContentFullWidth` in a separate PR.